controllers/neutronapi_controller.go

```diff
@@ -1513,6 +1513,22 @@ func (r *NeutronAPIReconciler) getTransportURL(
 	return string(transportURL), nil
 }
 
+func (r *NeutronAPIReconciler) isQuorumQueuesEnabled(
```

Comment:
@lmiccini should this be an helper function common to all operators? I'm wondering if this code can be centralized in infra-operator so that other services can access the same function in case we need it.
In addition, while I understand this is a very explicit check, I'm wondering if having:

```golang
transportURLQuorumQueues := string(transportURLSecret.Data["quorumqueues"]) == "true"
```
would be enough.

Reply:
nice catch, it's probably a leftover, let me try with the oneliner (should work but I don't trust myself)